packages/libs/react-ui/src/components/Divider/Divider.css.ts

```diff
@@ -1,15 +1,35 @@
-import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+import { token } from '../../styles';
 import { atoms } from '../../styles/atoms.css';
-import { tokens } from '../../styles/index';
 
-export const dividerClass = style([
-  atoms({
-    width: '100%',
-    marginBlock: 'lg',
-    border: 'none',
-  }),
-  {
-    backgroundColor: tokens.kda.foundation.color.border.base.bold,
-    height: tokens.kda.foundation.border.width.hairline,
+export const dividerClass = recipe({
+  base: [
+    atoms({
+      width: '100%',
+      marginBlock: 'lg',
+      border: 'none',
+    }),
+    {
+      height: token('border.width.hairline'),
+    },
+  ],
+  variants: {
+    variant: {
+      subtle: {
+        backgroundColor: token('color.border.base.subtle'),
+      },
+      base: {
+        backgroundColor: token('color.border.base.default'),
+      },
+      bold: {
+        backgroundColor: token('color.border.base.bold'),
+        boldest: {
+          backgroundColor: token('color.border.base.boldest'),
+        },
+        highContrast: {
+          backgroundColor: token('color.border.base.high-contrast'),
+        },
```

Comment:
I feel like these are at the wrong level?

Reply:
You mean that they should be in the atoms?

Reply:
nvm, I just noticed that I make a mistake while resolving Sams comment.